build.gradle

```diff
@@ -286,7 +283,7 @@ curseforge {
 
         mainArtifact(jar) {
             relations {
-                requiredLibrary "jeitweaker"
+                requiredDependency "jeitweaker"
```

Comment:
Does this work?

Reply:
https://github.com/matthewprenger/CurseGradle/blob/2167af2c8a8a7d781d5e792b422b3337d6b1dac8/src/main/groovy/com/matthewprenger/cursegradle/CurseRelation.groovy#L13-L19